pkg/apis/core/annotation_key_constants.go

```diff
@@ -155,4 +155,20 @@ const (
 	// heuristics will often populate topology hints on EndpointSlices, but that
 	// is not a requirement.
 	AnnotationTopologyMode = "service.kubernetes.io/topology-mode"
+
+	// MountOptionAnnotation can be used for a persisent volume claim and
+	// a persisent volume to specify the name of the storage class. This annotation has
+	// been deprecated in favor of the `storageClassName` field in the PVC and PV spec.
+	//
+	// Please note that it's currently still used and will be held for backwards compatibility.
+	// And this annotation takes precedence over the `storageClassName` field when both are set.
+	// It might be removed in a future release, see kubernetes/kubernetes#116336 for tracking.
+	BetaStorageClassAnnotation = "volume.beta.kubernetes.io/storage-class"
```

Comment:
incorrect comment ?

Reply:
Fixed: renamed to BetaStorageClassAnnotation